media-libs/netpbm/files/netpbm-10.86.21-build.patch

```diff
@@ -0,0 +1,66 @@
+- Remake netpbm-10.76.00-build.patch for 10.86.21; original description follows.
```

Comment:
don't refer to old patches.  just create the new one and update the description as needed.

Reply:
Done